### ISSUE TITLE
doesn't work on the Edge Browser

### ISSUE THREAD

**Issue by NicoleMayer** [WinTenDev/rminder#12]
Original I thought it's edge browser's bug about notification, but it seems that other websites can pop up the notification. Since microsoft edge is based on chrome, the notification mechanism may be similar. Could you help me find out the reason why the notification doesn't show up? Thank you very much!

**Comment by Kholid060** [comment]
I've tested in edge browser and it worked completely fine
![image](https://user-images.githubusercontent.com/22908993/121614566-88da2b00-ca91-11eb-80d5-f2c71d555b56.png)
Have you tried re-install the extension? and if you still have the same problem after re-install the extension, open the extension details, activate the developer mode, click the background page, and if you see an error in the console tab. Screenshot it and send it here

**Comment by NicoleMayer** [comment]
Sorry for my late reply. I re-install the extension several times and the notification still doesn't show up. I check the console and there is nothing here (Log level: All levels). An interesting thing I found is that if I check "Play notification sound", it does play. So I could use the sound to get the water time. I tested it on the Mac and the edge browser on the Mac is not so stable. I guess the problem comes from it.
